src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogHttpClient.java

```diff
@@ -567,10 +567,10 @@ private boolean postLogs(String payload) {
             rd.close();
 
             if ("{}".equals(result.toString())) {
-                logger.fine(String.format("Logs API call was sent successfully!"));
+                logger.fine(String.format("Webhook API call was sent successfully!"));
```

Comment:
Was this a typo?

Reply:
Yes, it was a typo I introduced in a previous PR, but it shouldn't be fixed here.

About this PR as a whole: The `recommended` label was added when logs would only work with an Agent, but now it works Agentless as well. I asked on slack and the main reason we would want to keep `recommended` there is because it might be more performant to send things through the Agent (because it does aggregation). So maybe I should just leave it as it is and close this PR...